### PR TITLE
Bug 8 has fixed, task 28 has done

### PR DESCRIPTION
**Pull request by universezzz**

### CODE REVIEW

**Review by slavasgod — APPROVED** [review]

**Review by OlegPanasyuk — APPROVED** [review]

File: api/controllers/auth.js [hunk 1]
```diff
@@ -50,54 +63,65 @@ const reset = async (req, res) => {
...
-    const UserId = await services.resetPasswordApprove({
+
+    const userId = await services.resetPasswordApprove({
       linkId: decodeURIComponent(linkId),
     });
-    if (UserId) {
+
+    if (userId) {
       res.status(200).send({
         status: 200,
```
Comment (melonholy) [inline api/controllers/auth.js:77]: Duplicated code @OlegPanasyuk

File: api/controllers/auth.js [hunk 2]
```diff
@@ -50,54 +63,65 @@ const reset = async (req, res) => {
...
   } catch (e) {
     throw new errors.ResetPasswordError(e.message);
   }
 };
 
 const changePassword = async (req, res) => {
-  const errors = validateAuth(req.body);
-  if (!isEmpty(errors)) {
-    return res.status(400).json(errors);
+  const { password } = req.body;
```
Comment (melonholy) [inline api/controllers/auth.js]: i offer you not to use dectructurization for 1 field

**Review by melonholy — CHANGES_REQUESTED** [review]

File: api/controllers/auth.js (hunk 1 above)
Comment (universezzz) [reply api/controllers/auth.js:77]: It'll be fixed, in next spring

File: api/controllers/auth.js (hunk 2 above)
Comment (universezzz) [reply api/controllers/auth.js]: Resolved

**Pull request merged**